.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 .idea
+
+portfolio/src/main/resources/
```

Comment:
maybe add the actual file to ignore instead? So you don't have to move `city-data.csv` to another directory :)